index.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 const alfy = require('alfy');
+const alfredNotifier = require('alfred-notifier');
```

Comment:
Should we bundle the notifier in Alfy? Any downsides doing that? We could add the ability to turn it off, but I don't see why anyone would want to do that.